Dockerfile

```diff
@@ -38,7 +38,10 @@ COPY ${SCOPE_CONFIG:-config.json} /app/config.json
 
 # install the app
 RUN cd /app && npm install
-RUN cd /app && npm run build
+
+# No need to run the build when apache is serving assets from the host (see docker-compose.yml)
+# RUN cd /app && npm run build
+
 RUN cd /app/opt && poetry install
 
```

Comment:
To be added after:
```
# install the app
RUN cd /app && npm install
```

```suggestion
# build static html
RUN cd /app && SCOPE_CONFIG=./config.json SCOPE_PORT=${SCOPE_PORT} npm run build
```

Reply:
I said this below, but I think the assets should be served from the host, unless you have a better reason. Otherwise you have to rebuild the image every time you change the config.